src/world_interface/kinematic_common_interface.cpp

```diff
@@ -180,14 +180,14 @@ void setJointMotorPower(robot::types::jointid_t joint, double power) {
 		float pitchPwr = power;
 		float rollPwr = getJointPowerValue(jointid_t::differentialRoll);
 		gearpos_t gearPwr = wristKinematics().jointPowerToGearPower(jointpos_t{pitchPwr, rollPwr});
-		setMotorPos(motorid_t::differentialLeft, gearPwr.left);
-		setMotorPos(motorid_t::differentialRight, gearPwr.right);
+		setMotorPower(motorid_t::differentialLeft, gearPwr.left);
+		setMotorPower(motorid_t::differentialRight, gearPwr.right);
 	} else if (joint == jointid_t::differentialRoll) {
 		float pitchPwr = getJointPowerValue(jointid_t::differentialPitch);
 		float rollPwr = power;
 		gearpos_t gearPwr = wristKinematics().jointPowerToGearPower(jointpos_t{pitchPwr, rollPwr});
-		setMotorPos(motorid_t::differentialLeft, gearPwr.left);
-		setMotorPos(motorid_t::differentialRight, gearPwr.right);
+		setMotorPower(motorid_t::differentialLeft, gearPwr.left);
+		setMotorPower(motorid_t::differentialRight, gearPwr.right);
```

Comment:
Good catch - totally missed this!

Reply:
Caused a little confusion when the differential wasn't moving but thankfully wasn't that bad to track down lol